quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -856,10 +856,12 @@ impl RecoveryOps for GRecovery {
         r.rtt() + (r.rttvar() * 4).max(GRANULARITY)
     }
 
+    /// The most recent data delivery rate estimate in bytes/s.
     fn delivery_rate(&self) -> u64 {
         self.pacer
             .bandwidth_estimate(&self.rtt_stats)
-            .to_bits_per_second()
+            .to_bits_per_second() /
+            8
```

Comment:
Please look at the implementation of `maybe_qlog` also.  It seems like the two congestion control implementations put different info in that field.  Don't know how to best reconcile the two.

Reply:
Yea not sure who is using qlog currently. I'll track this separately for now.